docs/notebooks/grids_cartesian.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<p>Maybe something like:</p><p>"Here, for example, we create a..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/963/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>